package.json

```diff
@@ -56,9 +56,9 @@
   },
   "husky": {
     "hooks": {
-      "post-checkout": "yarnhook",
-      "post-merge": "yarnhook",
-      "post-rewrite": "yarnhook",
+      "post-checkout": "npx --no-install yarnhook",
+      "post-merge": "npx --no-install yarnhook",
+      "post-rewrite": "npx --no-install yarnhook",
```

Comment:
I think there is no need to use `npx` here, only in the `.husky/*` files. This might be related to a mix of settings for husky v4 and v6. In the v4 you can use local packages directly, but not in the v6.

It is out of the scope of this issue, but we have settings for both versions but miss the `"prepare": "husky install"` part in the scripts section of package to avoid problems with newer versions on npm. Maybe is a good option to create a new issue to make a full migration to v6.

Reply:
Thanks for pointing this out! I read about the v4 and v6 migration a bit more because at first I was also confused on the definitions in `package.json`.
I not ran this thing: https://github.com/typicode/husky-4-to-6 (I'm guessing it didn't run in the previous migration).
I don't think it's unreasonable to also migrate in this PR, since you pointed it out and since it's straightforward :) less PRs to review :)